pages/index/IndexHandler.inc.php

```diff
@@ -113,8 +112,22 @@ function _displayPressIndexPage($press, &$templateMgr) {
 		}
 
 		// Display Featured Books
-		$displayFeaturedBooks = $press->getSetting('displayFeaturedBooks');
-		$templateMgr->assign('displayFeaturedBooks', $displayFeaturedBooks);
+		if ($press->getSetting('displayFeaturedBooks')) {
+			$featureDao = DAORegistry::getDAO('FeatureDAO');
+			$featuredMonographIds = $featureDao->getSequencesByAssoc(ASSOC_TYPE_PRESS, $press->getId());
+			if (!empty( $featuredMonographIds)) {
+				$featuredMonographs = array();
+				$publishedMonographDao = DAORegistry::getDAO('PublishedMonographDAO');
+				$publishedMonographs = $publishedMonographDao->getByPressId($press->getId());
+				$publishedMonographs = $publishedMonographs->toAssociativeArray();
+				foreach($featuredMonographIds as $key => $val) {
```

Comment:
Where will I find the current `Monograph` within the `while` loop? Is it assigned to a variable within `$publishedMonographs`, like `$publishedMonographs->current`?